internal.service.binding_bindingpathmaps.yaml

```diff
@@ -0,0 +1,146 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.3.0
+  creationTimestamp: null
+  name: bindingpathmaps.internal.service.binding
+spec:
+  group: internal.service.binding
+  names:
+    kind: BindingPathMap
+    listKind: BindingPathMapList
+    plural: bindingpathmaps
+    singular: bindingpathmap
+  scope: Namespaced
```

Comment:
We should strongly consider making this resource cluster scoped.

1. there isn't a lot of variance between namespaces
2. one less resource to manage across multiple namespace
3. this is behavior a cluster admin should probably take responsibility for. It's similar in scope to installing a new resource, or setting up RBAC for the binding controller to access the resource in the first place.